File: cprogramming/primeloop.c
#include <stdio.h>

int ctrlNUM, counter;

int main(void)
{
    scanf("%d", &ctrlNUM);
    for(int i = 1; i<=ctrlNUM; i++)
    {
        for(int j = 1; j<=i; j++)
        {
            if(i%j==0)
            counter++;
        }
        if(counter<=2)
        printf("%d", i);
    }
}

File: java/primeloop.java
import java.util.Scanner;
public class primeloop 
{
    static int ctrlNUM, counter;
    public static void main(String[] args)
    {
        Scanner scnr = new Scanner(System.in);
        ctrlNUM = scnr.nextInt();
        for(int i = 2; i<= ctrlNUM; i++)
        {
            for (int j = 2; j<= i; j++)
            {
                if(i%j==0)
                {
                   counter++;
                }
                
            }
            if (counter <= 2)
                {
                    System.out.println(i);
                }
                counter = 0;
        }
        scnr.close();
    }
    
}
